themes/compose/layouts/_default/baseof.html

```diff
@@ -16,7 +16,7 @@
   <header class="nav_header">
     {{- partial "nav" . }}
   </header>
-  <main class="main wrap pt-4">
+  <main class="main wrap md">
```

Comment:
I guess this line also changed every bulleted list to be mono-space font in the FAQ text body. The mixed fonts look a bit strange to me.